### PR TITLE
small update

### PR DESCRIPTION
**Pull request by rkapl123**
Hi Augusto!

I've fixed a wrong reference to excel-dna 1.0 in the Introduction and added a nice tutorial on C++/CLI code wrapper class with Excel-DNA that I found again.

-regards,
Roland

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ If you are a VBA developer interested in moving to .NET, you should start with [
 - As a comprehensive example using many of the Excel-DNA features, be inspired by the [Financial Analytics Suite (FinAnSu)](http://brymck.github.com/finansu/), an open-source C# add-in built by Bryan McKelvey.
 - [Ross McLean](https://web.archive.org/web/20140902002824/http://www.blog.methodsinexcel.co.uk/2010/08/16/why-excel-dna/) has a series of posts on getting started with Excel-DNA.
 - [Mikael Katajamäki shows how to use Microsoft Solver Foundation to build a curve fitting function for Excel](http://mikejuniperhill.blogspot.com/2013/06/using-ms-solver-foundation-and-c-in.html)
+- [Mikael Katajamäki shows how to use C++/CLI code as a wrapper class for native (Quantlib based) C++ code and interfaced the C# client code to Excel by using Excel-DNA] http://mikejuniperhill.blogspot.com/2018/10/wilmott-software-interoperability-in.html
```
Comment (augustoproiete) [inline index.md]:
```suggestion
- [Mikael Katajamäki shows how to use C++/CLI code as a wrapper class for native (Quantlib based) C++ code and interfaced the C# client code to Excel by using Excel-DNA](http://mikejuniperhill.blogspot.com/2018/10/wilmott-software-interoperability-in.html)
```

I think you need to add the parenthesis here to create the link

**Review by augustoproiete — CHANGES_REQUESTED** [review]
Thanks @rkapl123. Looks good, one minor thing on one of the hyperlinks

File: index.md (hunk 1 above)
Comment (rkapl123) [reply index.md]: Sorry, I updated it along with the other commit adding the missing references to wix installer and strong name.

**Review by augustoproiete — APPROVED** [review]
LGTM!

**Comment by augustoproiete** [comment]
@rkapl123 your changes have been merged, thanks for your contribution :+1:

**Pull request merged**